apps/storefront-e2e/src/support/page_fragments/search.fragment.ts

```diff
@@ -22,25 +22,25 @@ export class SearchFragment {
     this.getSearchResultsWrapper().contains('View all products');
 
   search = (text: string) => {
-    cy.intercept(
-      `**/catalog-search-suggestions?q=${encodeURIComponent(text).replace(
-        /%20/g,
-        '+'
-      )}*`
-    ).as('searchQuery');
+    const encoded = encodeURIComponent(text).replace(/%20/g, '+');
+
+    cy.intercept(`/catalog-search-suggestions?q=${encoded}*`).as('searchQuery');
 
     // trigger hydration
-    this.getTypeahead().click();
-    // wait for hydration
+    cy.intercept('/assets/box.component-*.js').as('searchHydationFiles');
```

Comment:
But we shouldn't rely on details of how assets are bundled.... and why this component has been chosen? :)

Reply:
Yep, that looks weird, but that's the only thing which proves that hydration was triggered 

I've just checked which one is being loaded all the time on search hydration + this one is the last one